src/modules/blocks/event-website/template.js

```diff
@@ -24,6 +24,8 @@ const { URLInput } = wpEditor;
  */
 
 const placeholder = __( 'Add Event Website', 'the-events-calendar' );
+const buttonPlaceholder = __( 'Button text', 'the-events-calendar' );
+const urlPlaceholder = __( 'website URL', 'the-events-calendar' );
```

Comment:
To be consistent with `Button text` in capitalization.
```suggestion
const urlPlaceholder = __( 'Website URL', 'the-events-calendar' );
```